build/optaplanner-build-parent/pom.xml

```diff
@@ -829,6 +862,7 @@
         </property>
       </activation>
       <properties>
+        <animal.sniffer.skip>true</animal.sniffer.skip>
```

Comment:
The `quickProfile` skips plugins that are enabled by default. If we make the Android API check a part of the `fullProfile`, we don't have to skip it here.

Reply:
Fixed